portal-web/test/functional/com/liferay/portalweb/macros/Navigator.macro

```diff
@@ -184,6 +184,8 @@ definition {
 
 	macro openNodePort {
 		Open(locator1 = "http://localhost:${nodePort}");
+
+		LiferayEvent.initializeLiferayEventLog();
```

Comment:
See https://github.com/john-co/liferay-portal/pull/293#pullrequestreview-408227052, let's avoid this and try to never ever destroy data in the process.

If the mechanisms we have aren't enough, let's try analyze the issue and come up with a solution that doesn't erase information.